src/hostapi/webaudio/README.txt

```diff
@@ -0,0 +1 @@
+pa_hostapi_skeleton.c provides a starting point for implementing support for a new host API with PortAudio. The idea is that you copy it to a new directory inside /hostapi and start editing.
```

Comment:
Please add a README.md file with detailed instructions for building, testing and using this new hostapi. That will help us test and review the code and will also make sure people are able to use this contribution.